packages/cli/src/oclif/commands/validate.js

```diff
@@ -9,10 +9,10 @@ const { validateApp } = require('../../utils/api');
 
 class ValidateCommand extends BaseCommand {
   async perform() {
-    this.log('Validating project locally.');
+    this.log('Validating project locally here.');
```

Comment:
is this change important?

Reply:
nope! was badely placed a logging/debug add. Will revert.